docs/decision-records/2023-07-17_usage_of_this_repository/README.md

```diff
@@ -0,0 +1,31 @@
+# Usage of this repository
+
+## Decision
+
+All documentation regarding specification and standards revolving around SSI and related topics should be hosted in this
+repository. No documentation regarding any particular implementation should be hosted there. Existing documents
+referring to implementation items will be deleted.
+
+Once concrete implementations exist, they will be linked in the SSI Docu repository.
```

Comment:
@stefan-ettl we should discuss this with Manti, he derived the Standard documents from here

Reply:
@OSchlienz this PR is open for 2 days already, do you think you could reach an accord until EOB today?

Reply:
@paullatzelsperger the architecture week is until tomorrow, not shure if I will get a response from Manti.

Reply:
@OSchlienz @paullatzelsperger Can we merge this to keep the pace and if there are issues we can always amend with a follow-up DR?

Reply:
Let's merge it for now please. We could apply changes afterwards if necessary.